commec/screeners/check_reg_path.py

```diff
@@ -207,14 +210,50 @@ def parse_taxonomy_hits(
 
                     # Collect unique species from both regulated and non-regulated
                     reg_species.extend(regulated["species"])
+
                     # JSON serialization requires int, not np.int64, hence the map()
                     reg_taxids.extend(map(str, regulated["subject tax ids"]))
                     non_reg_taxids.extend(map(str, non_regulated["subject tax ids"]))
 
+                    regulated_taxa.append(regulated)
+                    non_regulated_taxa.append(non_regulated)
+
                     # These are the old values, now we simply count the size of the regulated, and non_regulated taxid arrays.
                     #n_reg += (top_hits["regulated"][top_hits['q. start'] == region['q. start']] != False).sum()
                     #n_total += len(top_hits["regulated"][top_hits['q. start'] == region['q. start']])
 
+                logger.debug(regulated_taxa)
+                logger.debug(non_regulated_taxa)
```

Comment:
I know this is a draft, just adding that if we do want to leave this as a debug log, we probably want to add more context